<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>234</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>WBC Control</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>475</width>
        <height>232</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>214</red>
               <green>214</green>
               <blue>214</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>214</red>
               <green>214</green>
               <blue>214</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>214</red>
               <green>214</green>
               <blue>214</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>214</red>
               <green>214</green>
               <blue>214</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <layout class="QVBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">

QGroupBox {
    border: 2px solid    gray;
    border-radius: 0px;
    border-color: rgb(0,0,0);
    margin-top: 0.5em;
   background-color: hsv(0,0,220);
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: -10px 3px 0 3px;
}

</string>
            </property>
            <property name="title">
             <string>WBC</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QGroupBox" name="groupBoxWBCTasks">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="palette">
                    <palette>
                     <active>
                      <colorrole role="Button">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Base">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Window">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </active>
                     <inactive>
                      <colorrole role="Button">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Base">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Window">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </inactive>
                     <disabled>
                      <colorrole role="Button">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Base">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Window">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </disabled>
                    </palette>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                    </font>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">

QGroupBox {
    border: 3px solid    gray;
    border-radius: 9px;
    border-color: rgb(224, 0, 0);
    margin-top: 0.5em;
    background-color:  hsv(0, 0,255);
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: -10px 3px 0px 3px;
}

</string>
                   </property>
                   <property name="title">
                    <string>Constraints</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBoxSolver">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="palette">
                    <palette>
                     <active>
                      <colorrole role="Button">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Base">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Window">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </active>
                     <inactive>
                      <colorrole role="Button">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Base">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Window">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </inactive>
                     <disabled>
                      <colorrole role="Button">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Base">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="Window">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>255</green>
                         <blue>255</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </disabled>
                    </palette>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                    </font>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">

QGroupBox {
    border: 3px solid    gray;
    border-radius: 9px;
    border-color: rgb(0,200,0);
    margin-top: 0.5em;
   background-color: hsv(0,0,255);
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: -10px 3px 0 3px;
}

</string>
                   </property>
                   <property name="title">
                    <string>Solver</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QGroupBox" name="groupBox_2">
                   <property name="styleSheet">
                    <string notr="true">

QGroupBox {
    border: 1px solid    gray;
    border-radius: 0px;
    border-color: rgb(0,0,0);
    margin-top: 0.5em;
    background-color:  hsv(0, 0,255);
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: -10px 3px 0 3px;
}

</string>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QPushButton" name="bnImportConfig">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Import</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="bnExportConfig">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Export</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="bnReconfigureWBC">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Reconfigure WBC</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
